### DISCUSSION TITLE
CVE-2021-43798

### DISCUSSION THREAD

**Discussion by njohnson-q** [Security-Onion-Solutions/securityonion#6817] [Unsupported Versions] [answered]
Wanted to know if anyone can confirm that a fix is being implemented for this CVE or if it is relevant? It seems that the current version of security onion is vulnerable to this attack. The following versions are not affected: 8.0.7, 8.1.8, 8.2.7, or 8.3.1.

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/issues/6321

It will be upgraded in .100
